src/components/DetailsPods/detailsPods.util.js

```diff
@@ -0,0 +1,12 @@
+export const generatePods = (pods = {}) => {
+  return pods.podsList
+    ? pods.podsList.map(pod => {
+        return {
+          value: pod.name,
+          className: 'link',
+          status: pod.status,
+          pending: !!pods.podsPending.find(item => item.name === pod.name)
```

Comment:
1. Please prefer using `Boolean(value)` to convert a value to boolean, rather than `!!value`
2. You don't really need to convert to boolean. Instead of using the array method `find` you can use `some`. Both methods iterate the array and stop on the first item that satisfies the provided criterion (the function you pass). The difference is in their return value. `find` returns the item itself, if found, or `undefined` if none is found. `some` returns `true` if found, or `false` if none is found.

```suggestion
          pending: pods.podsPending.some(item => item.name === pod.name)
```